Cargo.toml

```diff
@@ -19,18 +19,86 @@ name = "inx-chronicle"
 path = "bin/inx-chronicle/src/main.rs"
 
 [dependencies]
+
+# Required
 async-recursion = { version = "1.0", default-features = false }
 async-trait = { version = "0.1", default-features = false }
 clap = { version = "3.1", default-features = false, features = [ "derive", "std" ] }
+derive_more = { version = "0.99", default-features = false }
 dotenv = { version = "0.15", default-features = false }
 env_logger = { version = "0.9", default-features = false, features = ["termcolor", "atty", "humantime"] }
 futures = { version = "0.3", default-features = false }
+hex = { version = "0.4", default-features = false }
 inx = { git = "https://github.com/iotaledger/inx", version = "0.2", default-features = false }
 log = { version = "0.4", default-features = false }
 mongodb = { version = "2.1", default-features = false, features = ["tokio-runtime"] }
 serde = { version = "1.0", features = ["derive"], default-features = false }
+serde_json = { version = "1.0", default-features = false, features = ["std"] }
 thiserror = { version = "1.0", default-features = false }
+time = { version = "0.3", default-features = false, features = ["std"] }
 tokio = { version = "1.17", default-features = false, features = [ "macros", "rt-multi-thread", "signal" ] }
 tokio-stream = { version = "0.1", default-features = false }
 toml = { version = "0.5", default-features = false }
+tonic = { version = "0.7", default-features = false }
```

Comment:
I'll add the missing types to the `inx` crate so that we don't need to import `tonic` directly.

Reply:
We need the tonic::transport::Error as well